stade/urls.py

```diff
@@ -18,3 +18,6 @@
     urlpatterns = [path('__debug__/', include(debug_toolbar.urls))] + urlpatterns
 
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+if hasattr(settings, 'JOIST_UPLOAD_STS_ARN'):
```

Comment:
This is fine for now, in the future though we'll want to start determining if we're in prod/dev/testing more directly. We already use `if settings.DEBUG` as a proxy for this in various places which has caused problems in the past. @brianhelba

Reply:
My concern with this approach is that when prod is misconfigured, we don't fail cleanly and obviously. I'm fine with 500 errors, but not unpredictable behavior.

Reply:
joist will complain if you enable it but don't have the proper settings it requires

Reply:
I'd suggest using `if settings.DEBUG` here, to make it easier to grep for in the future, unless this definitely breaks something now.